src/libs/getTL.js

```diff
@@ -0,0 +1,15 @@
+/* Data get information */
+export async function getDataTLFromId(text, city = false) {
```

Comment:
I renamed from `catch` to `get` and killed `catchCityTLFromId` as it can be expressed here directly.